lib/src/huge_listview_controller.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:flutter/material.dart';
+
+class HugeListViewController {
+  void Function()? onReload;
+  void Function(bool reloadPage)? onInvalidateList;
+  void Function(int)? setTotalItemCount;
+
+  set totalItemCount(int count) {
+    if (setTotalItemCount != null) {
```

Comment:
Not terribly important but I would normally use:

    if (callback == null) throw Exception("...");
    callback();

Less verbose and the compiler/IDE finds that `callback()` isn't null by itself, without `!`.

Reply:
yep, that's better for sure.